core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -124,6 +124,41 @@ class EntityService(protected val userInfo: UserInfo, val dataSource: SlickDataS
       }
     }
 
+  def renameEntityType(workspaceName: WorkspaceName, renameInfo: EntityTypeRename): Future[Int] = {
+    import org.broadinstitute.dsde.rawls.dataaccess.slick.{DataAccess, ReadAction}
```

Comment:
question, possibly to David since he suggested the syntax- what's our team approach to scope imports? Generally, I would look to avoid them for decluttering/readability, but I'm open to the idea that's helpful to import things right before they are used.